sponsor.html

```diff
@@ -106,7 +106,113 @@
+            event model—an event that is hosted both in-person and virtually, 
+            drawing on the success of our completely virtual event in 2020 and our 
+            history of in-person events. In addition to implementing this hybrid 
+            model, we are expanding to the west coast, creating a new venue location 
+            in San Francisco in order to increase our in-person event capacity and 
+            enable west coast hackers to participate.
+          </p>
+          <p>
+            Technica is not a typical hackathon—it serves as a place for the brightest 
+            thinkers in the country to come together to collaborate and share their 
+            innovative ideas. Our focus is on providing a welcoming, engaging, 
+            and creative environment to support <b>women and non-binary people</b> in tech 
+            culture. We pride ourselves in helping both beginners and experienced 
+            hackers explore technology and develop their skills. 
+          </p>
+          <p>
+            Not only does Technica serve as an incubator for creating and promoting 
+            a gender-diverse workforce in the future, but it also provides an optimal 
+            opportunity for recruitment and outreach. We would love for you to get 
+            involved as a sponsor for Technica 2021! 
+          </p>
+          <p>
+            Our available sponsorship tiers are listed on the following page. Additionally, 
+            we are happy to state that Technica has <b>501(c)(3) status.</b> Please let 
+            us know if you are interested in becoming a part of Technica, or if 
+            you have any questions or concerns about the information below. We look 
+            forward to hearing from you!
+          </p>
+          <p>
+            Thank you,
+            <br>
+            The Technica Team
+          </p>
+        </div>
+      </section>
+      
+    </article>
+  </div>
+
+  <!-- stats -->
+  <div class = "stats">
+    <div class="overall-stats">
+      <div class="heading">
+        Statistics
+      </div>
+      <div class="row">
+        <div class="col-lg-5">
+          <div class="row" style="margin-bottom: 30px">
+            <div class="col-6">
+              <span class="stat-number">850+</span>
+              <br>
+              <span class="stat-text">Attendees</span>
+            </div>
+            <div class="col-6">
+              <span class="stat-number">150+</span>
+              <br>
+              <span class="stat-text">Schools</span>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-6">
+              <span class="stat-number">144</span>
+              <br>
+              <span class="stat-text">Projects submitted</span>
+              </div>
+            <div class="col-6">
+              <span class="stat-number">37</span>
+              <br>
+              <span class="stat-text">Sponsors</span>
```

Comment:
let's update the stats for 2021 (can also view in design in figma):
- 1868 attendees
- 150+ schools
- 215 projects
- 35 sponsors

Reply:
can you also add 36 countries & 6 continents underneath these:')